src/main/java/org/commcare/session/CommCareSession.java

```diff
@@ -161,8 +143,7 @@ public CommCareSession(CommCareSession oldCommCareSession) {
         return entries;
     }
 
-    private Vector<Entry> getStillValidEntriesFromMenu(Menu menu,
-                                                       OrderedHashtable<String, String> currentSessionData) {
```

Comment:
`currentSessionData` was unused after the changes made in https://github.com/dimagi/commcare-core/pull/778/ but got left in by accident, so just fixing that now